### PR TITLE
added icons to SurvivalSelection and changed font and color

### PR DESCRIPTION
**Pull request by MatinMirzaei**

### CODE REVIEW

**Review by taerichard — APPROVED** [review]

File: calmTheStorm/calmTheStorm/SurvivalSelection.swift [hunk 1]
```diff
@@ -11,25 +11,37 @@ struct SurvivalSelection: View {
...
-            .offset( y:-70)
-        
+            .offset( y:-180)
+            
+            
+            HStack{
+                Image(systemName: "info.circle")
+                    .font(.system(size: 30))
+                    .offset( y:-50)
         Button("Survival Info"){
```
Comment (RJB888) [inline calmTheStorm/calmTheStorm/SurvivalSelection.swift:26]: Text color is not consistent with home page.  Please fix.

File: calmTheStorm/calmTheStorm/SurvivalSelection.swift [hunk 2]
```diff
@@ -11,25 +11,37 @@ struct SurvivalSelection: View {
...
-        .padding()
-        .cornerRadius(20)
+        .padding(3)
+        .offset( y:-50)
+            }
         
+            HStack{
+                Image(systemName: "map")
+                    .font(.system(size: 30))
         Button("Map Info"){
```
Comment (RJB888) [inline calmTheStorm/calmTheStorm/SurvivalSelection.swift:37]: Same as above.  Text color not consistent with home page.

**Review by RJB888 — CHANGES_REQUESTED** [review]

File: calmTheStorm/calmTheStorm/SurvivalSelection.swift (hunk 1 above)
Comment (MatinMirzaei) [reply calmTheStorm/calmTheStorm/SurvivalSelection.swift:26]: @RJB888  Text-color fixed

File: calmTheStorm/calmTheStorm/SurvivalSelection.swift (hunk 2 above)
Comment (MatinMirzaei) [reply calmTheStorm/calmTheStorm/SurvivalSelection.swift:38]: @RJB888 Updated

**Review by RJB888 — APPROVED** [review]
Looks good.

**Pull request merged**